Kotlin/demo/src/main/java/com/wolt/blurhashapp/MainActivity.kt

```diff
@@ -16,8 +17,11 @@ class MainActivity : AppCompatActivity() {
         val etInput: EditText = findViewById(R.id.etInput)
         val ivResult: ImageView = findViewById(R.id.ivResult)
         findViewById<View>(R.id.tvDecode).setOnClickListener {
```

Comment:
Why not replace the `findViewById`s in this file with Kotlin synthetics as well

Reply:
good idea, will do 👍